src/test/scala/is/hail/utils/BetterBlockMatrixSuite.scala

```diff
@@ -0,0 +1,46 @@
+package is.hail.utils
```

Comment:
Maybe this file shouldn't exist? I made it when there were no multiply tests to convince myself that it worked. The single test here could probably be moved to BlockMatrixIsDistributedMatrixSuite.scala